modules/openapi-generator/src/main/resources/python/api_client.handlebars

```diff
@@ -293,7 +293,7 @@ class StyleFormSerializer(ParameterSerializerBase):
         prefix_separator_iterator: typing.Optional[PrefixSeparatorIterator] = None
     ) -> str:
         if prefix_separator_iterator is None:
-            prefix_separator_iterator = PrefixSeparatorIterator('?', '&')
```

Comment:
Why have you changed the question mark string to empty string?
This eliminates the first ? when form parameters are sent in query strings

Reply:
Correct. As far as I can tell, the leading ? is not supposed to be sent for application/x-www-form-urlencoded form data. The initial ? separator is used to delimit the boundary between the URI of a queryable object, and a set of words used to express a query on that object.

Reply:
For example, see this page: https://developer.mozilla.org/en-US/docs/Learn/Forms/Sending_and_retrieving_form_data

Reply:
or this: https://www.oreilly.com/openbook/cgi/ch04_02.html

Reply:
How about instead setting `prefix_separator_iterator=PrefixSeparatorIterator('', '&')` in RequestBody.__serialize_application_x_www_form_data ?
That way this change is specific only to the needed content-type?

Reply:
Sure thing. I'll give that a try. Thanks!

Reply:
Never mind, defaulting to this looks okay as the query parameter invocation passes in the needed prefix_separator_iterator